### DISCUSSION TITLE
Preferences to Control UI Element Visibility

### DISCUSSION THREAD

**Discussion by kahn-msupply** [msupply-foundation/open-msupply#4294] [Ideas]
## Is your feature request related to a problem? Please describe 👀

**CERB:** https://cerb.msupply.org/profiles/ticket/MR-7694-TX/conversation

Some users / facilities may not want to use certain features that are available and would prefer if these preferences were hidden, rather than accessible (even without permission to use that feature).

For example, some facilities may not want to use Inbound & Outbound Returns. The wording may be causing confusion for some users also.

## Describe the solution you'd like 🎁

If the feature was hidden from a UI, then the user can focus on what they are allowed to access. It may also help with telling user via UI what permissions they have. 

### Describe alternatives you've considered 💭

I would like to open a discussion to options :)

My input, as we add more features to Open mSupply, our list of features available will also grow. It might be helpful for productivity / navigation to hide features for users / facilities who don't have permissions.

### Moneyworks Jobcode 🧰

~

**Comment by andreievg** [comment]
Triage, sounds like product owner design and decision. Converting to discussion

**Comment by adamdewey** [comment]
Totally agree with the user on this!
